addons/resource/models/resource_calendar.py

```diff
@@ -22,6 +22,16 @@
 from .utils import Intervals, float_to_time, make_aware, datetime_to_string, string_to_datetime
 from odoo.addons.hr_work_entry_contract.models.hr_work_intervals import WorkIntervals
 
+WEEKDAYS = {
+    '0': 'Monday',
+    '1': 'Tuesday',
+    '2': 'Wednesday',
+    '3': 'Thursday',
+    '4': 'Friday',
+    '5': 'Saturday',
+    '6': 'Sunday',
+}
```

Comment:
unnecessary for only one call + I believe this can be done more efficiently with the date/datetime format methods